Cargo.lock

```diff
@@ -1227,6 +1227,15 @@ version = "0.12.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8a9ee70c43aaf417c914396645a0fa852624801b24ebb7ae78fe8272889ac888"
 
+[[package]]
+name = "hashbrown"
+version = "0.13.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "43a3c133739dddd0d2990f9a4bdf8eb4b21ef50e4851ca85ab661199821d510e"
+dependencies = [
+ "ahash",
+]
```

Comment:
Can we update the dep so it's also using `0.14.0`?

Reply:
done!

Reply:
Still duplicated in the diff here :)

Reply:
now its fixed for real (and github is showing it too)